app/services/view/AutoCompleteItem.scala

```diff
@@ -43,5 +45,8 @@ object AutoCompleteItem {
     offices map (d => AutoCompleteItem(s"${d.description} - ${d.code}", d.code))
 
   def fromProcedureCodes(procedureCodes: List[ProcedureCode]): List[AutoCompleteItem] =
-    procedureCodes map (pc => AutoCompleteItem(s"${pc.code} - ${pc.description}", pc.code))
+    procedureCodes map (pc => AutoCompleteItem(formatProcedureCode(pc), pc.code))
+
+  def fromAdditionalProcedureCodes(additionalProcedureCodes: List[AdditionalProcedureCode]): List[AutoCompleteItem] =
```

Comment:
Personal opinion :smiling_face_with_three_hearts: 
`fromProcedureCodes` and `fromAdditionalProcedureCodes` should be in their respective service class.

Reply:
As already stated, service classes in this use case adds no value